components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -432,6 +443,16 @@ - (NSString *)accessibilityValue {
   return self.button.accessibilityValue;
 }
 
+#pragma mark - MDCInkViewDelegate
+
+-(void)inkAnimationDidEnd:(MDCInkView *)inkView {
+  self.inkAnimationInProgress = NO;
```

Comment:
Hmm, maybe in this case having one at a time ripples doesn't hurt anything because the button being touched to produce the ripple isn't really big enough for multiple fingers/touches

Reply:
Many of our other components support multiple simultaneous ripple animations.  I believe we should have a good design justification for deviating within Bottom Navigation.

Reply:
That's fair - is not particularly important.

Reply:
For historical context, the case I'm most concerned with is rapid touches feeling polished (i.e. animations don't flicker out of existence on subsequent touches).

Reply:
The delayed ripple is a client-blocking bug. I would think missing some touch feedback would also be pretty important.